.automation.conf/tempest/skip-lists/tempest-full

```diff
@@ -0,0 +1 @@
+tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_subnet_details.*: "Cirros image doesn't have '/var/run/udhcpc.eth0.pid"
```

Comment:
The skip list is probably fairly environment-specific. We might need to give it a multinode-specific name, and conditionally use it in config.sh.

Reply:
Still a problem